src/js/libcs/Operators.js

```diff
@@ -5097,3 +5097,256 @@ evaluator.removeelement$1 = function(args, modifs) {
     if (arg.ctype === "geo") removeElement(arg.value.name);
     else console.log("argument of removeelement is undefined or not of type <geo>");
 };
+
+
+evaluator.guess$1 = function(args, modifs) {
+    function pslq(x) {
+        const c = Math.PI * Math.E;
```

Comment:
I do not want to comment this line (Did the algorithm detect `pi*e` as rational? ;-))